hathor/manager.py

```diff
@@ -647,6 +662,17 @@ def _make_block_template(self, parent_block: Block, parent_txs: 'ParentTxs', cur
             score=sum_weights(parent_block_metadata.score, weight),
         )
 
+    def get_current_score(self, parent_block: Optional[Block] = None, timestamp: Optional[int] = None) -> float:
```

Comment:
I don't know how that would work. We either need the current score (and thus we have no block instance, so it would be a classmethod), or we need the score from a given parent block, and we have an instance (and thus it can't be a classmethod anymore). Do you have any suggestion? Maybe the signature isn't good, I'm not sure how this is being used it could be refactored in a different way, but I wouldn't make it goal to put it on the block class but rather make it clearer why it exists.

Reply:
I feel we could move it to the next PR since it is not being used in this PR.

Reply:
Moved to the next PR.